htdocs/cron/class/cronjob.class.php

```diff
@@ -1268,13 +1269,15 @@ public function run_jobs($userlogin)
 					dol_syslog(get_class($this)."::run_jobs END result=".$result." error=".$errmsg, LOG_ERR);
 
 					$this->error = $errmsg;
-					$this->lastoutput = (!empty($object->output) ? $object->output."\n" : "").$errmsg;
+					if (!empty($object->output) && $this::MAXIMUM_LENGTH_FOR_LASTOUTPUT_FIELD > mb_strlen(trim($object->output."\n".$errmsg))) $this->lastoutput = $object->output."\n".$errmsg;
```

Comment:
Why not just:

`$this->lastoutput = dol_trunc(!empty($object->output) ? $object->output."\n" : "").$errmsg, $this::MAXIMUM_LENGTH_FOR_LASTOUTPUT_FIELD, 'right', 'UTF-8', 1);`